sources/gov/kaunas/municipality/educational_institutions.yaml

```diff
@@ -0,0 +1,30 @@
+id: "gov/kaunas/municipality/educational_institutions"
+title: "Educational institutions"
+description: "CSV file containing data about Kaunas educational institutions"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
+  - "csv:"
+provider: "gov/kaunas/municipality"
```

Comment:
This provider does not exist in `providers/` directory. But also this is fault of my script for not checking it properly.